src/ports/postgres/modules/deep_learning/madlib_keras_wrapper.py_in

```diff
@@ -325,8 +326,19 @@ def compile_model(model, compile_params, custom_function_map=None):
     optimizers = get_optimizers()
     (opt_name,final_args,compile_dict) = parse_and_validate_compile_params(compile_params)
     if custom_function_map is not None:
-        map=dill.loads(custom_function_map)
-        compile_dict['loss']=map[compile_dict['loss']]
+        local_map=dill.loads(custom_function_map)
+
+        compile_dict['loss']=local_map[compile_dict['loss']] \
+            if compile_dict['loss'] in local_map else compile_dict['loss']
+
+        new_metrics = []
+        for i in compile_dict['metrics']:
```

Comment:
Just a note, we currently support only 1 metrics, so we don't need to iterate here, the loop is fine too. Will leave it upto you, if you want to change it, else this is good too.

Reply:
I know that the `compile_dict['metrics']` is an array of a single element but in case we support multiple metrics this code would be useful so I prefer leaving it as is.